TERMINOLOGY.md

```diff
@@ -1,53 +1,70 @@
 
-## Randomized predictors
+- _Protected attribute_. The term is based on anti-discrimination laws that define specific _protected classes_. Since we seek to apply group fairness in a wider range of settings, we avoid this terminology.
 
-Rather than providing deterministic output many predictors produced by disparity mitigation techniques in the fairness literature are randomized. As a consequence, it is possible to get different output from the predictor's `predict` method when providing samples with identical features.
+#### Parity constraints
 
-## Group Fairness
+Group fairness is typically formalized by a set of constraints on the behavior of the predictor called **parity constraints** (also called criteria). Parity constraints require that some aspect (or aspects) of the predictor behavior be comparable across the groups defined by group-membership features.
 
-### Protected or sensitive attribute, group, or data
+Let _X_ denote a feature vector used for predictions, _A_ be a single group-membership feature (such as age or race), and _Y_ be the true label. Parity constraints are phrased in terms of expectations with respect to the distribution over (_X,A,Y_).
+For example, in fairlearn, we consider the following types of parity constraints.
 
-In the scientific literature concerning fairness one usually selects a single feature or multiple features of a dataset as a protected or sensitive set of attributes. The objective is to optimize a chosen disparity metric (for a definition see one of the later sections) based on these attributes. In some scenarios these attributes may also be held out from the actual training data (note that holding out data is not sufficient for fairness due to the myriad of correlations in real datasets). Overall, every combination of the attributes defines what we refer to as a group.
-There is no definitive standard yet, but for this repository, we shall use the variable name `group_data` for a matrix of these attributes. This choice provides a few advantages:
+_Binary classification_:
 
-- The term ‘protected attribute’ has a specific meaning in legal contexts which does not always match the data science usage of the term
-- It may signify the presence of multiple sensitive attributes (as a matrix of values), as opposed to other choices such as `sensitive_attribute`.
-- It does not falsely imply ordering between `X`, `y`, and `group_data`, as opposed to, for example, `Z`.
-- It does not falsely imply that it is always part of the training features, as opposed to variable names that subscript `X`.
+- _Statistical parity_ (also known as _demographic parity_): A classifier _h_ satisfies statistical parity under a distribution over (_X, A, Y_) if its prediction _h_(_X_) is statistically independent of the group membership _A_. This is equivalent to E[_h_(_X_) | _A_=_a_] = E[_h_(_X_)] for all _a_. [[Agarwal et al.]](https://arxiv.org/pdf/1803.02453.pdf)
```

Comment:
Are we supposed to rename demographic parity to statistical parity? I don't mind doing that, just want to use the right term.

Reply:
Let me ask around if there's a sentiment that one is better than the other.

Reply:
resolving to demographic parity